vignettes/orderly.Rmd

```diff
@@ -722,7 +722,23 @@ orderly::orderly_run(name,
 
 ## Using environment variables and secrets
 
-Envirionment variables that are stored in the top-level `orderly_envir.yml` are (since orderly 1.1.6) available as environment variables, via `Sys.getenv()`.  For example, if your `orderly_envir.yml` contains
+Environment variables declared in `orderly.yml` are made available in the report script (since orderly 1.1.11). For example, if your `orderly.yml` contains
+
+```yaml
+environment_variables:
+  external_file: EXTERNAL_FILE_PATH
```

Comment:
note that this does not match the `$` prefixing that you use above, but this way is better :stuck_out_tongue:

Reply:
Thanks - this was what I wanted to use. Forgot that I hadn't tided that up in the test.